### ISSUE TITLE
Numbering inconsistency for Jatakas

### ISSUE THREAD

**Issue by sabbamitta** [suttacentral/bilara-data#1106]
At least for some Jatakas the numbering differ between English and German. Checking for Ja542 for example, EN has the Maha-Ummagga Jataka, DE has the Khandahala Jataka. Root in this case has "Umaṅgajātaka", which agrees with EN.

The correct number for DE Ja542 seems to be 544, the Candakumārajātaka.

There may be more cases of that sort.

German Jataka blurbs agree with the numbering of German Jatakas, so in case numbers need to be changed, the numbers of the blurbs should undergo the same change.

**Comment by sabbamitta** [comment]
List of required changes:
- DE Ja542 should be numbered 544 (incl. blurb)
- DE Ja544 should be numbered 545 (incl. blurb)
- DE Ja545 should be numbered 546 (incl. blurb)
- DE Ja546 should be numbered 542 (incl. blurb)

More to follow …

**Comment by sabbamitta** [comment]
All Jatakas mentioned here do have blurbs meanwhile (but for some reason I can't edit the above post).

**Comment by sujato** [comment]
I believe I have made the fixes to these, please check them.

You'll have to publish the blurbs on Bilara.

**Comment by sabbamitta** [comment]
What you did seems to be correct. Thank you! I have now adjusted the blurbs. 

Are the changes meant to be on the website already? I don't see them there yet.

I expected to find more of such cases, but haven't so far. I'm however not yet finished with the Jatakas. You've now closed the issue, so I'll make a new one in case I still find something. Maybe that was it.

**Comment by sabbamitta** [comment]
Bhante, I have to reopen this issue. Following [this post](https://discourse.suttacentral.net/t/on-the-very-idea-of-an-article-about-the-pali-canon/26578/13), I wanted to add a remark about this Bible parallel to the blurb of the Umanga Jataka, but didn't find the blurb in the expected place in the Bilara file. Looking on the website, the Jataka is still found as Ja 546, while it should be 542. It seems the same is true for the other ones too.

I don't know what I checked last time, but it's unfortunately not fixed.

**Adding:**
So, I now checked again, and it is indeed correct now. What I found wrong last time was due to cache issues. Waiting for a while and a few other page refreshs finally show the correct Jatakas with the correct numbers. I only had to move the blurbs around, which I did now. 

So I think I can close the issue again now, hopefully for good. :heavy_check_mark:
